Adyen/Integration/02_ManageAdyenConnection.md

```diff
@@ -0,0 +1,161 @@
+    
+7. Enable the ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *MIT fraud check* toggle if you do not trust the Adyen&trade; fraud check and want to do your own. This might be relevant if there is a longer period between the Adyen&trade; fraud check and the authorization receipt. 
+
+8. Click the [SAVE] button.   
+   The connection will be created. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+    The data are synchronized between Actindo and Adyen&trade;. The view for creating connections is automatically closed when the connection has been created and synchronized. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+
+    ![Credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionSettings.png "[Credentials]")
+
+
+
+## Edit Adyen connection
+
+For detailed information, see [Edit PSP connection](../../Payments/Integration/01_ManageConnection.md#edit-psp-connection) in the *Payments* documentation.
+
+
+
+## Configure Adyen connection
+
+Configure the Adyen&trade; connection after you have created it. In the *Payments* connection settings, you can enter the event codes that are to be ignored. Alternatively, you can select the unwanted event codes in the Adyen&trade; webhook settings. 
+
+The Adyen&trade; API has two endpoints, one for a synchronous and one for an asynchronous processing. If you decide for asynchronous processing, what is recommended, you can configure the number of workers to process messages in asynchronous processing. For detailed information, see [Define synchronous or asynchronous processing](./01_ManageAdyenSettings.md#define-synchronous-or-asynchronous-processing).
+
+
+#### Prerequisites
+
+In the *Actindo Core1 Platform*, you are permitted to create connections in *Payments*.
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+ 
+ ![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+ 1. Click the Adyen&trade; connection you want to edit in the list of connections.   
+   The *Edit connection* view is displayed. By default, the *Credentials* tab is selected.
+
+    ![Edit Adyen settings](../../Assets/Screenshots/Adyen/Integration/EditCredentials.png "[Edit Adyen credentials]")
+
+2. Click the *Settings* tab.   
+  The *Ignored event codes* setting is displayed by default.
+
+   ![Ignored event codes](../../Assets/Screenshots/Adyen/Integration/IgnoredEventCode.png "[Ignored event codes]")
+
+3. If desired, enter an event code to be ignored in the *Key* field and in the *Value* field as well. Note that you can also define the event codes to be ignored in the Adyen&trade; backend. For detailed information, see [Create webhook in Adyen](./01_ManageAdyenSettings.md#create-adyen-webhook).
+    The event code to be ignored is entered. 
```

Comment:
Vorschlag: If desired, enter an event code to be ignored both in the *Key* and in the *Value* fields. Note that you can also define the event codes to be ignored in the Adyen&trade; backend. For detailed information, see [Create webhook in Adyen](./01_ManageAdyenSettings.md#create-adyen-webhook).

The event code to be ignored is entered... --> Ich würde den Satz hier weglassen (selbstverständlich, oder?).